src/compiler/crystal/config.cr

```diff
@@ -34,8 +34,40 @@ module Crystal
       Time.unix(time).to_s("%Y-%m-%d")
     end
 
-    def self.default_target_triple
-      {{env("CRYSTAL_CONFIG_TARGET")}} || LLVM.default_target_triple
+    @@default_target : Crystal::Codegen::Target?
+
+    def self.default_target : Crystal::Codegen::Target
```

Comment:
(optional) This can be written as

```crystal
getter(default_target : Crystal::Codegen::Target) do 
  # code here
end
```

Reply:
I'm not particularly drawn to a specific style here, but I like being able to grep for `def self.default_target`.